README.md

```diff
@@ -0,0 +1,41 @@
+# Hansken Kaitai Plugin Template for Python
+
+This repository contains a template for a Hansken Kaitai plugin written in Python.
+The template contains the general structure of a Kaitai plugin, including the required build steps.
+
+To get started developing Hansken Extraction Plugins, read the [Getting Started Documentation](https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/dev/python/getting_started.html).
+Furthermore, it is strongly recommended to take note of the [Hansken Extraction Plugins for plugin developers documentation](https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/).
+
+The implementer of this template, is required to conduct a few general steps to enable parsing the contents of a specific filetype to a JSON, as is explained in more detail below.
+The template contains utility in [`kaitai_utils.py`](kaitai_utils.py). This utility python file, which only contains 'static' methods, contains the generic parse methods. 
+The implementer of the new Kaitai plugin, does not need to touch this `kaitai_utils.py` file. 
+
+For more information on the supported formats by Kaitai, go to https://formats.kaitai.io/. 
+
+
+To transform this template into an implementation, we suggest to conduct the following steps:
+* Clone the template plugin to get started on implementing your Kaitai plugin for Hansken
+* Navigate to https://ide.kaitai.io/#
+* Download the python parser for the filetype of your interest to [`structs`](structs)
```

Comment:
I think we need to make clear here that _all_ generated Python files need to be copied, not just the parser file. With the Zip parser, for example, Kaitai also generates dos_datetime.py which is required by the actual parser.

Reply:
agree